dialog-el.html

```diff
@@ -355,7 +357,9 @@
       }.bind(this));
     }
 
-    proto.close = function() {
+    proto.close = function(e) {
+      // Allow persistent dialogs if the element clicked on has an open-dialog attribute.
+      if (e && e.target && e.target.attributes["open-dialog"]) return;
```

Comment:
It sounds like you just need to prevent a click event from bubbling, do you have a JSBin with your issue I could look at?